.github/workflows/build-nix.yml

```diff
@@ -8,15 +8,15 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.4.0
+    - uses: actions/checkout@v3
```

Comment:
Really good job updating all of our `workflows` to use newer actions, but you missed our `actions`. In the `.github/actions` folder there are still some `.yml` files which are not updated - containing things like `actions/cache@v2`, etc and these are also tringgering the `save-state` warning we are trying to get rid off here: https://github.com/onekey-sec/unblob/actions/runs/4125774689

Reply:
Good catch ! Will take care of it.

Reply:
Done.